cmd/oras/discover.go

```diff
@@ -174,27 +173,14 @@ func printDiscoveredReferrersTable(refs []artifactspec.Descriptor, verbose bool)
 }
 
 // printDiscoveredReferrersJSON prints referrer list in JSON equivalent to the
-// API result: https://github.com/oras-project/artifacts-spec/blob/v1.0.0-rc.1/manifest-referrers-api.md#artifact-referrers-api-results
-func printDiscoveredReferrersJSON(desc ocispec.Descriptor, refs []artifactspec.Descriptor) error {
-	type referrerDesc struct {
-		Digest    digest.Digest `json:"digest"`
-		MediaType string        `json:"mediaType"`
-		Artifact  string        `json:"artifactType"`
-		Size      int64         `json:"size"`
-	}
-	output := struct {
-		Referrers []referrerDesc `json:"referrers"`
-	}{
-		Referrers: make([]referrerDesc, len(refs)),
-	}
-
-	for i, ref := range refs {
-		output.Referrers[i] = referrerDesc{
-			Digest:    ref.Digest,
-			Artifact:  ref.ArtifactType,
-			Size:      ref.Size,
-			MediaType: ref.MediaType,
-		}
+// image index: https://github.com/opencontainers/image-spec/blob/main/image-index.md#image-index-property-descriptions
```

Comment:
Could you use a fixed tag instead of `main` in the URL?

Reply:
updated